src/Processors/QtiV2/Out/ResponseProcessingBuilders/QtiResponseProcessingBuilder.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace LearnosityQti\Processors\QtiV2\Out\ResponseProcessingBuilders;
+
+use qtism\common\enums\BaseType;
+use qtism\data\processing\ResponseProcessing;
+use qtism\data\rules\SetOutcomeValue;
+use qtism\data\rules\ResponseRuleCollection;
+use qtism\data\rules\ResponseRule;
+use qtism\data\rules\ResponseCondition;
+use qtism\data\rules\ResponseIf;
+use qtism\data\rules\ResponseElseIf;
+use qtism\data\rules\ResponseElse;
+use qtism\data\expressions\operators\Match;
+use qtism\data\expressions\Variable;
+use qtism\data\expressions\BaseValue;
+use qtism\data\expressions\Correct;
+use qtism\data\expressions\ExpressionCollection;
+
+class QtiResponseProcessingBuilder
+{
+
+    public function build($score)
+    {
+        // creating feedback outcome
+        $feedbackResponseComponent = new SetOutcomeValue('FEEDBACK', new Variable('RESPONSE'));
+        $responseRuleCollection = new ResponseRuleCollection();
+        $responseRuleCollection->attach($feedbackResponseComponent);
+
+        // creating responseIf condition
+        $responseIfexpressionCollection = new ExpressionCollection();
```

Comment:
Style: Apply capitalization of camel case variable names consistently: i.e. `$responseIfExpressionCollection` instead of `$responseIfexpressionCollection`

Reply:
Done